src/app/pivot/pivot.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Pivot } from "@dhx/trial-pivot";
+import { getData } from "./data";
+
+import {
+	Component,
+	ElementRef,
+	OnInit,
+	OnDestroy,
+	ViewChild,
+	ViewEncapsulation,
+} from "@angular/core";
+
+@Component({
+	encapsulation: ViewEncapsulation.None,
+	selector: "pivot",
+	styleUrls: ["./pivot.component.css"],
+	template: `<div #container class="widget"></div>`,
+})
+
+export class PivotComponent implements OnInit, OnDestroy {
+	@ViewChild("container", { static: true }) pivot_container!: ElementRef;
+
+	private _table!: Pivot;
+
+	ngOnInit() {
+		const { dataset, fields } = getData();
```

Comment:
Formatting issue: the code uses both tabs and spaces for indentation, which causes misalignment.